app/src/components/layouts/Header.tsx

```diff
@@ -100,44 +126,263 @@ const DropdownPage = ({
+                setIsLanguageTab(true);
+                setIsLeftMenu(false);
+              }}
+              css={{ w: 260, h: 50 }}
+              icon={<GlobalIcon />}
+            >
+              Language
+            </Button>
+          </Grid>
+          <Grid css={{ margin: '10px 0 0 10px', width: 280, minHeight: 65, display: 'block' }}>
+            <Button
+              flat
+              color="primary"
+              size="md"
+              onClick={() => {
+                setIsDisplayTab(true);
+                setIsLeftMenu(false);
+              }}
+              css={{ w: 260, h: 50 }}
+            >
+              Display
+            </Button>
+          </Grid>
+          <Grid css={{ margin: '10px 0 0 10px', width: 280, minHeight: 65, display: 'block' }}>
+            {user ? (
+              <Button flat color="error" size="md" css={{ w: 260, h: 50 }}>
+                <Link to="/sign-out">
+                  <Text color="error">Log out</Text>
+                </Link>
+              </Button>
+            ) : (
+              <Button flat color="primary" size="md" css={{ w: 260, h: 50 }}>
+                <Link to="/sign-up">Sign Up</Link>
+              </Button>
             )}
-          </NavLink>
-        </Row>
-      ))}
-    </Grid.Container>
+          </Grid>
+        </Grid.Container>
+      </motion.div>
+      <motion.div>
+        <Grid.Container
+          css={{
+            flexDirection: 'column',
+          }}
+        >
+          {isLanguageTab && (
+            <>
+              <Grid css={{ margin: '10px 0 0 10px', width: 280, minHeight: 65, display: 'block' }}>
+                <Button
+                  light
+                  color="primary"
+                  size="md"
+                  onClick={() => {
+                    setIsLanguageTab(false);
+                    setIsLeftMenu(true);
+                  }}
+                  css={{ w: 260, h: 50 }}
+                  icon={
+                    <Player
+                      src={arrowLeft}
+                      hover
+                      autoplay={false}
+                      speed={0.75}
+                      className="w-8 h-8"
+                      loop
+                    />
+                  }
+                >
+                  Language
+                </Button>
+              </Grid>
+              {languages.map((lng) => (
+                <Grid
+                  key={lng}
+                  css={{ margin: '10px 0 0 10px', width: 280, minHeight: 65, display: 'block' }}
+                >
+                  <Button
+                    flat
+                    color="primary"
+                    size="md"
+                    onClick={() => {
+                      setIsLanguageTab(false);
+                      setIsLeftMenu(true);
+                      navigate(`/?lng=${lng}`);
+                    }}
+                    css={{ w: 260, h: 50 }}
+                  >
+                    {lng}
```

Comment:
chỗ này này, dùng { t(lng) }

Reply:
@ctuanle k bt beta 10 tụi sửa chi bên component link, h kiểu hấn toàn load lại trang khi click
t mới bỏ hết trong header với tab đồ, còn bên left drawer chắc t sửa sau tại hấn bị mất design luôn rồi